libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ExcelExporter.java

```diff
@@ -96,7 +97,11 @@ private void fillRow(Row row, List<String> values, CellStyle style) {
         }
         for (int column = 0; column < helper.getColumns(); column++) {
             Cell cell = row.createCell(column);
-            cell.setCellValue(values.get(column));
+            if(values.get(column) != null && (values.get(column).length() >= SpreadsheetVersion.EXCEL2007.getMaxTextLength())) {
+                cell.setCellValue("cell has exceeded max number of characters");
```

Comment:
it would be better visible in the generated spreadsheet if the would be a leading hashmark like "#cell has exceed may number of characters"

Reply:
Thanks for the suggestion @mcjaeger . I have made the suggested changes.